colab/README.md

```diff
@@ -37,6 +37,14 @@ model, pre-processes it for import, then compiles it using IREE
 
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/iree/blob/main/colab/tensorflow_hub_import.ipynb)
 
+### [tflite_text_classificaiton\.ipynb](tflite_text_classification.ipynb)
```

Comment:
(In the future, not necessarily on this PR)

Let's also link to this from https://google.github.io/iree/ml-frameworks/tensorflow-lite/ as part of https://github.com/google/iree/issues/5455